beacon-chain/core/blocks/block_operations.go

```diff
@@ -825,22 +802,8 @@ func VerifyIndexedAttestation(ctx context.Context, beaconState *stateTrie.Beacon
 			pubkeys = append(pubkeys, pk)
 		}
 	}
+	return attestationutil.VerifyIndexedAttestation(ctx, indexedAtt, pubkeys, beaconState.GenesisValidatorRoot(), beaconState.Fork())
```

Comment:
This method now diverges from the spec, it checks the pubkeys first then sorts the attesting indices,etc. Lets keep it the same.

Reply:
fixed. thanks